docs/source/nn/index.rst

```diff
@@ -0,0 +1,26 @@
+.. _nn-index:
+
+torch.nn
+===================================
+
+.. toctree::
+   :maxdepth: 1
+
+   nn_parameters.rst
+   nn_containers.rst
+   nn_convolution_layers.rst
+   nn_pooling_layers.rst
+   nn_padding_layers.rst
+   nn_non_linear_activations.rst
+   nn_non_linear_activations_other.rst
+   nn_normalization_layers .rst
+   nn_recurrent_layers.rst
+   nn_transformer_layers.rst
+   nn_linear_layers.rst
+   nn_dropout_layers.rst
+   nn_sparse_layers.rst
+   nn_distance_functions.rst
+   nn_loss_functions.rst
+   nn_vision_layers.rst
+   nn_dataparallel_layers.rst
+   nn_utilities.rst
```

Comment:
I'm not sure what is going on, but all of this looks like it is being repeated:

![image](https://user-images.githubusercontent.com/5652049/76989432-b54dd880-691c-11ea-8d4e-1097fd18e6cc.png)